### DISCUSSION TITLE
How to get Height and width of created landing page?

### DISCUSSION THREAD

**Discussion by throne1986** [GrapesJS/grapesjs#4122] [Q&A]
Hii guys

How do I get Height and width of a created landing page?

**Comment by throne1986** [comment]
@artf can you help out with this?

**Reply by artf** [reply]
Try to check the canvas API https://grapesjs.com/docs/api/canvas.html#canvas

**Reply by throne1986** [reply]
Hi I tried getRect()  but everytime I get the same width and height !!,  is there any way to get the real height and width when creating landing page?, thx
